lint.whitelist

```diff
@@ -481,6 +481,10 @@ TRAILING WHITESPACE: css/vendor-imports/mozilla/mozilla-central-reftests/css21/p
 TRAILING WHITESPACE: css/vendor-imports/mozilla/mozilla-central-reftests/flexbox/flexbox-mbp-horiz-001-rtl-reverse.xhtml
 TRAILING WHITESPACE: css/vendor-imports/mozilla/mozilla-central-reftests/multicol3/moz-multicol3-column-balancing-break-inside-avoid-1.html
 TRAILING WHITESPACE: css/vendor-imports/mozilla/mozilla-central-reftests/multicol3/moz-multicol3-column-balancing-break-inside-avoid-1-ref.html
+TRAILING WHITESPACE: css/css-text-3/white-space/textarea-pre-wrap-011.html
```

Comment:
Can you use HTML entities at the end of the line instead of whitelisting it? It makes it much easier to read the test (v. trying to read invisible characters!).